docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -82,9 +84,48 @@
 }
 ----
 
+==== Kubernetes Client ObjectMapper customization
+
+The Fabric8 Kubernetes Client uses its own `ObjectMapper` instance for serialization and deserialization of Kubernetes resources.
+This mapper is provided to the client through a `KubernetesSerialization` instance that's injected into
+the `KubernetesClient` bean.
+
+If for some reason you must customize the default `ObjectMapper` bean provided by this extension and used by the Kubernetes Client, you can do so by declaring a bean of type `@KubernetesClientObjectMapperCustomizer`.
```

Comment:
The beans need to implement the `KubernetesClientObjectMapperCustomizer` interface, not be annotated.

Reply:
Fixed.

:facepalm: good catch. Yesterday was a long day, I hope I didn't put in any other stupid mistake.